articles/install-guide/update.md

```diff
@@ -85,7 +131,7 @@ The update steps depend on your development environment. Choose your environment
 
 1. You can now open an existing Jupyter notebook and run it with the updated QDK.
```

Comment:
It's probably worth it to ask them to call `%workspace reload` from conda, to make sure caches are cleared up.

Reply:
we aren't shipping conda this release though, no?

Reply:
I meant Jupyter. Doh.